package.json

```diff
@@ -8,6 +8,19 @@
     "icons"
   ],
   "main": "index.js",
+  "exports": {
+    ".": {
+      "module": "./index.js",
+      "import": "./index.js",
+      "default": "./index.js"
+    },
+    "./icons": {
+      "module": "./icons.mjs",
+      "import": "./icons.mjs",
+      "default": "./icons.js"
+    }
+  },
+  "sideEffects": false,
```

Comment:
Did you test the behaviour if you omit this field? Is it still as expected?

Reply:
Yes, I tested and the tree shaking does not happen without it, webpack requires it. Read more [here](https://webpack.js.org/guides/tree-shaking/#mark-the-file-as-side-effect-free)